tests/integration/docusaurus/connecting_to_your_data/database/util.py

```diff
@@ -5,12 +5,14 @@ def load_data_into_database(
     import sqlalchemy as sa
 
     engine = sa.create_engine(connection_string)
-    engine.execute(f"DROP TABLE IF EXISTS {table_name}")
-    print(f"Dropping table {table_name}")
-    df = pd.read_csv(csv_path)
-    df = df.head(
-        10
-    )  # <WILL> This line is here to address performance issues we have been running into with cloud resources (ie redshift). Can be taken out
-    print(f"Creating table {table_name} from {csv_path}")
-    df.to_sql(name=table_name, con=engine, index=False)
-    engine.dispose()
+    connection = engine.connect()
+    try:
+        connection.execute(f"DROP TABLE IF EXISTS {table_name}")
+        df = pd.read_csv(csv_path)
+        df = df.head(
+            10
+        )  # <WILL> This line is here to address performance issues we have been running into with cloud resources (ie redshift). Can be taken out
+        df.to_sql(name=table_name, con=engine, index=False)
+    finally:
+        connection.close()
+        engine.dispose()
```

Comment:
`finally` is a great solution here!

A few questions:
1. Should we be removing the print statement in L14?
2. Should we use `except` as part of this flow? What kinds of errors can we run into with `connection.execute` and should we ensure that we catch those errors and deal with them appropriately?

Reply:
1. Yes, I can the note can be made much more concise (something like "# helping performance")
2. Great idea. I did some research, and `SQLAlchemyError` is the most general error that can happen, and is used throughout our codebase already. There are also examples in the code where we do an `except:` and a `raise` immediately afterwards.  What are your thoughts on something like:
    ```
except SQLAlchemyError as e:
        # TODO Add logging messages here
        raise
```

Reply:
Oh sorry I mean L14 in the previous version (before changes). We printed something out to the user and I was wondering if it should still be there now. I'm happy with your current note :)

And yes I like that! What should we `raise` though? This doesn't necessarily fit the scope of a `GreatExpectationsError` does it? I'm not familiar with its usage.

Reply:
1. Ah got it :) Re-added the print statements. 
2. Yes, I think this would fall outside of the scope of a `GreatExpectationsError` which are used more for errors in GE operation like if we can't find an `ExpectationSuite` etc.  I was thinking we could re-raise the same SqlAlchemy error but with an error message to the logger (kindof like : `ERROR - SOMETHING VERY BAD HAPPENED` below), so that we could preserve the Stacktrace and also have information about how "far" we got in the script 


![Screen Shot 2021-07-26 at 5 00 03 PM](https://user-images.githubusercontent.com/25670697/127074224-311e23af-b79d-44e5-978d-e4963d839aec.png)

Reply:
I like that! Logging something about the connection failing and raising the same error sounds like a viable solution to me!